entities/communication/voiceCallSetting.json

```diff
@@ -0,0 +1,94 @@
+{
+    "type": "object",
+    "properties": {
+        "setting_uid": {
+            "type": "string",
+            "description": "The voice call settings unique identifier."
+        },
+        "staff_uid": {
+            "type": "string",
+            "description": "Receiving forwarded call staff uid."
+        },
+        "forward_number": {
+            "type": "string",
+            "description": "Staff receiving forwarded call phone number."
+        },
+        "forwarding_enabled": {
+            "type": "boolean",
+            "description": "Is forwarding the voice call enabled or not."
+        },
+        "availability_uid": {
+            "type": "string",
+            "description": "Staff availability uid."
+        },
+        "forward_call": {
+            "type": "enum",
+            "values": [
+                "ALWAYS",
+                "WORK_HOURS",
+                "NEVER"],
+            "description": "Forward call option based on staff availability."
```

Comment:
```suggestion
        "call_forwarding_schedule": {
            "type": "enum",
            "values": [
                "ALWAYS",
                "WORK_HOURS",
                "NEVER"],
            "description": "When to forward the call based on staff availability."
```

Reply:
@eliran-vcita , Can be staff_forwarding_availability more accurate? as we are storing availability of the staff, 24/7 - based on working-hours -  not available to receive calls.

Reply:
Is this the flow?
call comes in => check `call_forwarding_schedule`
if `ALWAYS` => forward call to staff (no matter what their availability is)
if `WORK_HOURS` => forward call based on availability
if `NEVER` => don't forward?

Also - note to @alonyacobi - let's settle on a convention for enum values. I like this one with uppercase and snakecase values.

Reply:
Indeed @eliran-vcita , that's why i an not sure about schedule, maybe even call_forwarding_preference?

Reply:
preference is another word for setting 😄 

if this is the flow - I recommend `call_forwarding_policy`